### PR TITLE
Update tutorial Part 1 to MV3

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR makes the following changes:
* **README** for the repo
   * PR: https://github.com/mikehoffms/MicrosoftEdge-Extensions/blob/user/mikehoffms/tut-part-1/README.md
   * Before/Live: https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/README.md
   * Writer/Editor pass.
   * Added table of samples & their articles.
   * Updated link text to match latest destination title.
      * Changed [here] link text to destination link text.
      * Corrected the way articles are linked to.
   * Applied corp style guide.
      * Clarified.  
         * "Community resources" means the Code, Issues, and Discussions tabs of this repo.
         * add-ons = extensions.
   * Fixed typos.
* **Part 1 sample**
   * Copied manifest.json code listing from article https://learn.microsoft.com/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?tabs=v3#step-3-open-a-default-pop-up-dialog (from PR 3277's version of the article).
      * Changed `"manifest_version": 2,` to `"manifest_version": 3,`
      * Changed key name from `"browser_action": {` to:  `"action": {`
      * Changed capitalization to match the article.
      * Changed `name` string to be different from Part 2.
         * To avoid displaying identical names for the two very different completed demos in the Extensions list & in the Manage Extensions page.
         * When 1 & 2 are both installed, need a way to tell which is which, in the Extensions list.
      * Changed `description` string to be different than Part 2, and described the type of functionality.
         * To prevent identical descriptions for the two very different completed demos in the Manage Extensions page.
         * When 1 & 2 are both installed, need a way to tell which is which, in the Manage Extensions page..
         * The two demos are very different and need distinctive descriptions of the different functionality.
   * These changes have been tested; installed and ran this modified demo from local dir branch of this repo; demo ran ok, per screenshots of it.
* **Part 2 sample**
   * Copied Part 2 code in text files from Part 2 article per PR 3277 over the existing listings.  
      * To make leading spaces same between article & samples repo.
      * To update casing/ capitalization within strings.
      * To have a differentiated manifest "name" and "description" field.
   * In `manifest.json`, changed `description` string to be different than Part 1, and described the type of functionality.
   * These changes have been tested; installed and ran this modified demo from local dir branch of this repo; demo ran ok, per screenshots of it.

Related PR:
https://github.com/MicrosoftDocs/edge-developer/pull/3277

AB#54220337

### CODE REVIEW

**Comment by mikehoffms** [comment]
@bhuvanapriyap pls review.  
To approve the PR: **Files changed** page > **Review changes** green button > **Comment** option button along with a comment, "LGTM - PM".

**Review by bhuvanapriyap — APPROVED** [review]
Approved - LGTM - PM

**Review by bhuvanapriyap — APPROVED** [review]

**Review by mikehoffms — COMMENTED** [review]
LGTM - Writer/Editor

File: Extension samples/extension-getting-started-part1/part1/manifest.json
Comment (mikehoffms) [inline Extension samples/extension-getting-started-part1/part1/manifest.json:1]: it is an error in the demo (in extensions repo & docs) that the Description is same for manifest file in Part 1 & Part 2.  The Part 1 demo is very different (static, lacks JS) than Part 2 demo.  Poor UX: cannot tell which installed demo is Part 1 vs 2: 
image
Need to change Description in manifest file in /part1/ vs 2 dir of ext repo, and in article - in these PRs or future PRs.  
Part 1 desc: Demo of simple single static picture (simple demo); 
Part 2 desc: Demo of rotating images plural (JS demo).

Comment (mikehoffms) [reply Extension samples/extension-getting-started-part1/part1/manifest.json:1]: differentiated Name and Description, in manifests for Part 1 & 2, in extensions repo sample code & in Docs repo articles.

**Review by captainbrosset — APPROVED** [review]
I'm approving the code changes. They help make things clearer, and easier to differentiate the part 1 and part 2 extension samples. It's also very important to move to MV3, since that's what articles on learn.microsoft.com document.

**Review by bhuvanapriyap — APPROVED** [review]

**Review by yvsubhash — APPROVED** [review]
LGTM - Dev

**Pull request merged**